config/locales/client.en.yml

```diff
@@ -173,6 +174,8 @@ en:
     select_position: 'Select position'
     statement_placeholder: 'Share your thoughts with the group.'
     aria_statement_label: 'Optionally enter a statement to be recorded with your position'
+    messages:
+      created: Vote added
```

Comment:
For consistencies sake, can we make a habit of putting quotes around yaml messages?